lib/rspec/sidecar/app_availability.rb

```diff
@@ -0,0 +1,16 @@
+require 'zk'
+require 'rest_client'
+
+module RSpec::Sidecar::AppAvailability
+  def app_is_available
```

Comment:
in ruby conventions, you should require a block argument and end it in a question mark (since it returns a boolean)

```ruby
def app_is_available?(&blk)
```